client/js/components/vip.js

```diff
@@ -61,7 +61,7 @@ var VIP = React.createClass({
 
   mixins: [StoreWatchMixin(getInitialState)],
 
-  componentWillUpdate: function(nextProps, nextState) { //TODO: bug/does not show snackbar with 
+  componentWillUpdate: function(nextProps, nextState) { //TODO: bug/does not show snackbar with
```

Comment:
@csreyes : what's the TODO comment about? Looks like it got cut off mid-sentence.